x-pack/plugins/canvas/common/lib/index.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export * from './datatable';
```

Comment:
I know why you added this file, but nobody else probably does. At the very least, you should include a comment here to explain why it's required.

Also, this is *really* fragile. If we add anything else to `common/lib` (*highly* likely), there's a high probability that we'll forget to add it to this file, and now things can break silently and in very strange ways. If this is our only option, we need some tooling to enforce that we add new files here. It would be a lot better if we didn't have to deal with this at all though... 

Is there a way to force webpack to include a glob without having an explicit `import`?

Reply:
It was just to create a consistent index exporting for the `common/lib`. Even if you fail to add something later, our build will fail on CI complaining about a dependency that was not able to be found